application/server/server.go

```diff
@@ -190,10 +191,19 @@ func initNetworkAccessHeaders(n networking.NetworkAccess) {
 	))
 }
 
+func logIntegration(params lsp.InitializeParams) {
+	initParams := params.InitializationOptions
+	log.Info().Msg("IDE: " + params.ClientInfo.Name + "/" + params.ClientInfo.Version)
+	if initParams.IntegrationName != "" && initParams.IntegrationVersion != "" {
+		log.Info().Msg("snyk-plugin: " + initParams.IntegrationName + "/" + initParams.IntegrationVersion)
+	}
+}
+
 func initializeHandler(srv *jrpc2.Server, c *config.Config) handler.Func {
 	return handler.New(func(ctx context.Context, params lsp.InitializeParams) (any, error) {
 		method := "initializeHandler"
-		log.Info().Str("method", method).Interface("params", params).Msg("RECEIVING")
+		logIntegration(params) // Ideally this would be logged in the "initialized" handler, but the parameters are not available there
```

Comment:
Suggestion: Assuming that initializedHandler() comes after initializeHandler(), how about moving logIntegration() down in initializedHandler(), so that the information is logged as one block?